### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chatterjee-priyanka**
Update README indexing a few newly published explainers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,26 +16,40 @@ Sandbox's client-side APIs.
...
 
 * [Key/Value service trust model](key_value_service_trust_model.md)
-* [Key/Value service user-defined functions (UDFs)](key_value_user_defined_functions.md)
+* [Key/Value service use cases](key_value_service_use_cases.md)
+* [Key/Value service user-defined functions (UDFs)](key_value_service_user_defined_functions.md)
 * [Key/Value sharding](key_value_sharding.md)
 
+## Ad Retrieval service
+
+* [Protected App Siganls: Ad Retrieval service](ad_retrieval_overview.md)
```
Comment (salmanmlk) [inline README.md:37]: This link seems to be broken.

File: README.md [hunk 2]
```diff
@@ -16,26 +16,40 @@ Sandbox's client-side APIs.
...
 * [Bidding and Auction services](bidding_auction_services_api.md)
+* [Bidding and Auction services: Protected App Signals Ad Selection](bidding_auction_services_protected_app_signals.md)
 * [Bidding and Auction services onboarding and self-serve guide](bidding_auction_services_onboarding_self_serve_guide.md)
 * [Bidding and Auction services payload optimization](https://github.com/privacysandbox/fledge-docs/blob/main/bidding-auction-services-payload-optimization.md)
 * [Bidding and Auction services system design](bidding_auction_services_system_design.md)
 * [Bidding and Auction services multi seller auctions](https://github.com/privacysandbox/fledge-docs/blob/main/bidding_auction_services_multi_seller_auctions.md)
-* [Bidding and Auction services AWS cloud support and deployment guide](bidding_auction_services_aws_guide.md)
-* [Bidding and Auction services GCP cloud support and deployment guide](bidding_auction_services_gcp_guide.md)
 * [Protected Audience event level reporting with Bidding and Auction services](bidding_auction_event_level_reporting.md)
+* Cloud support: [Bidding and Auction services AWS cloud support and deployment guide](bidding_auction_services_aws_guide.md)
```
Comment (akshaypundle) [inline README.md:25]: Can we have 1 bullet for cloud support, with 2 items under it?

File: README.md [hunk 3]
```diff
@@ -16,26 +16,40 @@ Sandbox's client-side APIs.
...
+* [Key/Value service user-defined functions (UDFs)](key_value_service_user_defined_functions.md)
 * [Key/Value sharding](key_value_sharding.md)
 
+## Ad Retrieval service
+
+* [Protected App Siganls: Ad Retrieval service](ad_retrieval_overview.md)
+
+## Inference
+
+* Bidding and Auction services: Inference Overview](inference_overview.md)
```
Comment (akshaypundle) [inline README.md:41]: missing [  (so the link does not render as a link / work)

File: README.md (hunk 1 above)
Comment (akshaypundle) [reply README.md:37]: +1

**Review by akshaypundle — COMMENTED** [review]
Can we also add a link to PAS overview here?

**Comment by chatterjee-priyanka** [comment]
> Can we also add a link to PAS overview here?

I updated overall README for Protected Auctions from "Protected Audience", and explained what is Protected Auction that it includes both Protected Audience and Protected App Signals.
Linked in the client side explainers for those products.

Please review.

File: README.md (hunk 3 above)
Comment (chatterjee-priyanka) [reply README.md:41]: Fixed.

File: README.md (hunk 2 above)
Comment (chatterjee-priyanka) [reply README.md:25]: Sure, done.

File: README.md (hunk 1 above)
Comment (chatterjee-priyanka) [reply README.md]: Fixed.

File: README.md [hunk 4]
```diff
@@ -1,46 +1,67 @@
...
+    * AWS: [Bidding and Auction services AWS cloud support and deployment guide](bidding_auction_services_aws_guide.md)
+    * GCP: [Bidding and Auction services GCP cloud support and deployment guide](bidding_auction_services_gcp_guide.md)
 
 ## Key/Value service
 
 * [Key/Value service trust model](key_value_service_trust_model.md)
-* [Key/Value service user-defined functions (UDFs)](key_value_user_defined_functions.md)
+* [Key/Value service use cases](key_value_service_use_cases.md)
+* [Key/Value service user-defined functions (UDFs)](key_value_service_user_defined_functions.md)
 * [Key/Value sharding](key_value_sharding.md)
```
Comment (salmanmlk) [inline README.md:36]: This file seems to have a different name now.

File: README.md [hunk 5]
```diff
@@ -1,46 +1,67 @@
...
+* [Bidding and Auction services: Cost](bidding_auction_cost.md)
+
 ## Client and server integration
 
+### Protected Audience
 * [Browser - Bidding and Auction services integration](https://github.com/WICG/turtledove/blob/main/FLEDGE_browser_bidding_and_auction_API.md)
 * [Android - Bidding and Auction services integration high level document](https://developer.android.com/design-for-safety/privacy-sandbox/protected-audience-bidding-and-auction-services)
 
+### Protected App Signals
+* [Android: Protected App Signals]([Protected App Signals](https://developer.android.com/design-for-safety/privacy-sandbox/protected-app-signals)
```
Comment (salmanmlk) [inline README.md:63]: 1. This doesn't render well. 
2. Can use hyphen after Android to be consistent with PA. Following would be better:

[Android - Protected App Signals](https://developer.android.com/design-for-safety/privacy-sandbox/protected-app-signals)

File: README.md (hunk 4 above)
Comment (chatterjee-priyanka) [reply README.md]: Yeah, the KV sharding link changed. Updated here.

File: README.md (hunk 5 above)
Comment (chatterjee-priyanka) [reply README.md]: Fixed, thanks.

**Review by akshaypundle — COMMENTED** [review]
LGTM

**Pull request merged**